dependencies.gradle

```diff
@@ -3,46 +3,48 @@ ext {
             // Build tools and SDK
             buildTools      : "29.0.3",
             compileSdk      : 29,
+            gradle          : "3.6.1",
```

Comment:
I think a better name would be "agp" or something more explicit. "gradle" isn't android specific and is a bit vague

Reply:
@elihart both `agp` and `gradlePlugin` would be more descriptive.